From c30e0647c613d5bbcd3af2060b2c95520e33fd52 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Mon, 25 Jan 2021 16:03:15 +0200
Subject: [PATCH] [SW-30229] ppboot: goya2: increase size

This patch increase ppboot size by 4KB as current size of
16KB is exhausted.

Change-Id: I21699233c3639eea6e92d2900ef2f3110e10b5a3
---

diff --git a/goya2/common.h b/goya2/common.h
index 247c9bb..be5ae0f 100644
--- a/goya2/common.h
+++ b/goya2/common.h
@@ -10,7 +10,7 @@
 #include <goya2/goya2.h>
 #include <bitops.h>
 
-#define PPBOOT_SIZE			0x4000		/* 16KB */
+#define PPBOOT_SIZE			0x5000		/* 20KB */
 #define PREBOOT_SIZE			0xC000		/* 48KB */
 
 #define SRAM_SIZE			0x8000000	/* 128MB */
